docs/leidraad/security/richtlijn-applicatie-security-tools.md

```diff
@@ -27,7 +42,8 @@ De volgende doelgroepen kunnen met security tools aan de slag:
 
 **Security engineers** beoordelen de risico's.
 
-De **Chief Information Security Officer** (CISO) of andere verantwoordelijke bestuurder houdt de statistieken over de openstaande risico's in de gaten.
+De **Chief Information Security Officer** (CISO) of andere verantwoordelijke
+bestuurder houdt de statistieken over de openstaande risico's in de gaten.
```

Comment:
Zou er geen format check op de markdown moeten draaien in de pijplijn zodat dit soort reformat acties niet nodig zijn/kunnen gebeuren?

Reply:
Ja. Er zit nu een config in https://github.com/developer-overheid-nl/don-site/tree/main/.vscode, maar dat werkt alleen als men vscode gebruikt. Een markdown linter in de pijplijn staat nog op de backlog.